conversion/conversion.go

```diff
@@ -73,6 +73,9 @@ const (
 	// DYNAMODB is the driver name for AWS DynamoDB.
 	// This is an experimental driver; implementation in progress.
 	DYNAMODB string = "dynamodb"
+	// Target db for which schema is being generated
+	TARGET_SPANNER  string = "spanner"
+	TARGET_SPANGRES string = "spangres"
```

Comment:
Nit: Go comments start with first letter capital and end with a full stop. Suggest a full stop.
Also can we add a newline to separate the drivers and target db consts?